libobs/obs-audio-controls.c

```diff
@@ -16,6 +16,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include <math.h>
+#include <xmmintrin.h> 
```

Comment:
Trailing whitespace.  Need to be careful about that.  Can happen to the best of us though.